app/models/client.rb

```diff
@@ -759,6 +759,536 @@ def self.sort_options(show_vispdat: false, show_assessment: false)
     ]
   end
 
+  def self.prioritized_columns_data
+    @prioritized_columns_data ||= {
+      veteran: {
+        title: 'Veteran',
+        description: nil,
+        type: 'Boolean',
+      },
+      chronic_homeless: {
+        title: 'Chronically Homeless',
+        alt_title: 'Chronic',
```

Comment:
Pretty sure we can just use the title.
```suggestion
        title: 'Chronically Homeless',
```